src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/intave/SpeedIntave14.kt

```diff
@@ -36,36 +39,28 @@ import net.minecraft.entity.MovementType
  * @author larryngton
  */
 class SpeedIntave14(override val parent: ChoiceConfigurable<*>) : SpeedBHopBase("Intave14", parent) {
-
-
     companion object {
         private const val BOOST_CONSTANT = 0.003
     }
 
     private inner class Strafe(parent: EventListener) : ToggleableConfigurable(parent, "Strafe", true) {
-
-        private val strength by float("Strength", 0.29f, 0.01f..0.29f)
+        private val strength by float("Strength", 0.27f, 0.01f..0.27f)
```

Comment:
We may want to use a fixed value here, as this is a anti-cheat specific bypass. Strength will be included as 0.29f in every major config, essentially breaking the mode.